x/ccv/provider/keeper/proposal.go

```diff
@@ -263,20 +267,26 @@ func (k Keeper) MakeConsumerGenesis(
 	}
 
 	if prop.Top_N > 0 {
+		// get the consensus active validators
```

Comment:
Add more to the comment on why we need the active here, similar to your comment on the bonded validators below. I guess the reasoning here is that you do not want to "force" inactive validators to validate (?)